.github/workflows/test.yml

```diff
@@ -0,0 +1,10 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: PHPStan
+        uses: "docker://oskarstark/phpstan-ga"
```

Comment:
Try this one:
```yaml
on: [push, pull_request]

name: CI
jobs:
    composer:
        name: Composer checks
        runs-on: ubuntu-latest
        steps:
            - name: "Checkout"
              uses: actions/checkout@master

            - name: "Install dependencies with composer"
              run: php7.3 $(which composer) install --no-interaction --no-progress --no-suggest --no-scripts

            - name: "Validate composer.json"
              run:  $(which composer) validate

    phpstan:
        name: PHPStan
        runs-on: ubuntu-latest
        steps:
            - name: "Checkout"
              uses: actions/checkout@master

            - name: "Install dependencies with composer"
              run: php7.3 $(which composer) install --no-interaction --no-progress --no-suggest --no-scripts

            - name: "Static code analysis"
              run: $(which composer) phpstan
```

but we need to `require --dev phpstan/phpstan` in our composer.json

The problem with my action are extensions and some other. this should work out of the box

Reply:
Or add a composer script `composer phpstan` which does `vendor/bin/phpstan -c phpstan -c phpstan.neon.dist`, so could reuse it local.

and then in the GA: `run: $(which composer) phpstan`